tests/test_data/theglutenfreeaustrian.com/theglutenfreeaustrian_1.json

```diff
@@ -0,0 +1,53 @@
+{
+  "author": "Daniela Weiner",
+  "canonical_url": "https://theglutenfreeaustrian.com/gluten-free-sourdough-starter/",
+  "site_name": "The Gluten Free Austrian",
+  "host": "theglutenfreeaustrian.com",
+  "language": "en-US",
+  "title": "Gluten Free Sourdough Starter",
+  "ingredients": [
+    "350 grams gluten-free flour of choice",
+    "Water, as needed, filtered or bottled spring water"
+  ],
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "350 grams gluten-free flour of choice",
+        "Water, as needed, filtered or bottled spring water"
+      ],
+      "purpose": "Gluten Free Sourdough Starter"
+    }
+  ],
```

Comment:
Let's try to find test coverage for a recipe with more than one ingredient group, when testing support for them?  It's tricky to tell whether this genuinely confirms that the scraper supports extraction of groupings.

Reply:
Ah; there is suitable test coverage in the second JSON test data file for this scraper.  In that case: I think we can remove this `ingredient_groups` key?

```suggestion
```